website/docs/r/backup_recovery.html.markdown

```diff
@@ -0,0 +1,981 @@
+* `retrieve_archive_tasks` - (List) Specifies the list of persistent state of a retrieve of an archive task.
+Nested schema for **retrieve_archive_tasks**:
+	* `task_uid` - (String) Specifies the globally unique id for this retrieval of an archive task.
+	  * Constraints: The value must match regular expression `/^\\d+:\\d+:\\d+$/`.
+	* `uptier_expiry_times` - (List) Specifies how much time the retrieved entity is present in the hot-tiers.
+* `start_time_usecs` - (Integer) Specifies the start time of the Recovery in Unix timestamp epoch in microseconds.
+* `status` - (String) Status of the Recovery. 'Running' indicates that the Recovery is still running. 'Canceled' indicates that the Recovery has been cancelled. 'Canceling' indicates that the Recovery is in the process of being cancelled. 'Failed' indicates that the Recovery has failed. 'Succeeded' indicates that the Recovery has finished successfully. 'SucceededWithWarning' indicates that the Recovery finished successfully, but there were some warning messages. 'Skipped' indicates that the Recovery task was skipped.
+  * Constraints: Allowable values are: `Accepted`, `Running`, `Canceled`, `Canceling`, `Failed`, `Missed`, `Succeeded`, `SucceededWithWarning`, `OnHold`, `Finalizing`, `Skipped`, `LegalHold`.
+* `tear_down_message` - (String) Specifies the error message about the tear down operation if it fails.
+* `tear_down_status` - (String) Specifies the status of the tear down operation. This is only set when the canTearDown is set to true. 'DestroyScheduled' indicates that the tear down is ready to schedule. 'Destroying' indicates that the tear down is still running. 'Destroyed' indicates that the tear down succeeded. 'DestroyError' indicates that the tear down failed.
+  * Constraints: Allowable values are: `DestroyScheduled`, `Destroying`, `Destroyed`, `DestroyError`.
+
+
+## Import
+
+You can import the `ibm_backup_recovery` resource by using `id`. Specifies the id of the Recovery.The ID is formed using tenantID and resourceId.
+`id = <tenantId>::<recovery_id>`. 
+
+
+#### Syntax
+```
+import {
+	to = <ibm_backup_recovery_resource>
+	id = "<tenantId>:<recovery_id>"
```

Comment:
here we have single colon but in resources we are spiting by ("::") 
document the separator correctly most of resource goes with ("/") or (":") any specific reason for ("::")

Reply:
yes @hkantare 
The values of <tenantId> generally contains a '/' (like "jhxqx715r9/") and a lot resource Ids like recoveryId, policyId, groupIds etc. contains ":" (ex. a policy Id generally looks like this "5170815044477768:1732541085048:15126")
Since both '/' and ":" are present in tenantId and resourceId values, we have used "::"

Reply:
Also, I'll update `id = "<tenantId>:` to `id = "<tenantId>::` in documentation